webrender/src/render_backend.rs

```diff
@@ -267,8 +275,12 @@ impl Document {
                 &mut resource_profile.texture_cache,
                 &mut resource_profile.gpu_cache,
                 &self.dynamic_properties,
+                &mut self.clip_data_store,
             );
-            self.hit_tester = Some(frame_builder.create_hit_tester(&self.clip_scroll_tree));
+            self.hit_tester = Some(frame_builder.create_hit_tester(
```

Comment:
My only worry is creating a dependency between frame building and building the hit tester. We really need to be able to build the latter independently. Here we are first mutably passing the clip data store to the frame builder and then using it to build the hit tester.
Upon inspecting [the code that mutates the data store](https://github.com/servo/webrender/pull/3075/files#diff-a1e61c23c68078e351d27b526664bf56R449), it looks like only GPU thingies are modified so it should be safe to create the hit tester independently, but I'd feel better if we moved the line that creates the hit tester above the `FrameBuilder::build` call to make sure dependencies don't creep in.

Reply:
Yep, sounds good.